### PR TITLE
Use a custom modal component

### PR DESCRIPTION
**Pull request by dgdavid**
## Problem

There is quite some _duplication_ while building an _overview popup_

## Solution

To use a custom _Popup_ component built on top of PF4/Modal that allows using some default settings.

## Testing

* Added unit test
* Tested manually

## Screenshots

There are no changes at the UI level. Everything remains as it was. See it anyway:

| Before | After |
|-|-|
| popup-before | popup-after |

## Notes for reviewers

~~Real changes are only the last two commits. Other changes you can see are actually related to #133 since this PR was created on top of the branch used there. We can either, adapt this PR or mark #133 as ready and merge it first.~~ 

^^^ Already _fixed_.

### CODE REVIEW

**Comment by dgdavid** [comment]
> * Overall coverage decreased (**-2.5%**) to **80.204%**

@imobachgs, 

I do not understand why _coveralls_ says that. I even unsure if this is accurate.

**Comment by imobachgs** [comment]
> 
> @imobachgs,
> 
> I do not understand why _coveralls_ says that. I even unsure if this is accurate.

It might be related to #146.

**Comment by dgdavid** [comment]
> > @imobachgs,
> > I do not understand why _coveralls_ says that. I even unsure if this is accurate.
> 
> It might be related to #146.

Yes, it was. Fixed, thank you!

> > @imobachgs,
> > I do not understand why _coveralls_ says that. I even unsure if this is accurate.
> 
> It might be related to #146.

I think so, thanks you. It should be fixed, although there still something wrong in the _coveralls_ side. You can check https://coveralls.io/builds/48533480, where it says coverage has increased... or https://coveralls.io/builds/48533486 where it says coverage has decreased.

:man_shrugging:

File: web/src/FirstUser.jsx [hunk 1]
```diff
@@ -87,27 +87,15 @@ export default function Users() {
...
-            Cancel
-          </Button>,
-          <Button key="remove" variant="link" onClick={remove} isDisabled={!userIsDefined}>
-            Do not create a user
-          </Button>
-        ]}
+        onConfirm={accept}
+        confirmDisabled={formValues.userName === ""}
+        onCancel={cancel}
+        onUnset={remove}
```
[the remark this answers is not in the document]
Comment (dgdavid) [reply web/src/FirstUser.jsx:96]: See https://github.com/yast/d-installer/pull/144/commits/a6b8f2626d07d752412ed9a5f89c450b71fc25ad

We can discuss it tomorrow. Switching PR to draft mode.

**Review by imobachgs — APPROVED** [review]

**Pull request merged**